libs/analytics/src/lib/components/pie-chart/pie-chart.component.ts

```diff
@@ -72,6 +72,13 @@ export class PieChartComponent {
     this.chart?.updateOptions(this.pieChartOptions);
   }
 
-  private previousSelection?: string;
+  previousSelection?: string;
   @Output() selection: EventEmitter<string> = new EventEmitter();
+
+  toggleSelect() {
+    const index = this.pieChartOptions.labels.findIndex(label => label === this.previousSelection);
+    if (index >= 0) {
+      this.chart.toggleDataPointSelection(index);
+    }
```

Comment:
If not found, index is always -1
```suggestion
    if (index !== -1) this.chart.toggleDataPointSelection(index);
```